src/secrets/CommandCat.ts

```diff
@@ -80,28 +84,41 @@ class CommandGet extends CommandPolykey {
         }
         const hasErrored = await binUtils.retryAuthentication(async (auth) => {
           // Write secret paths to input stream
-          const response = await pkClient.rpcClient.methods.vaultsSecretsGet();
+          const response = await pkClient.rpcClient.methods.vaultsSecretsCat();
           const writer = response.writable.getWriter();
-          let first = true;
           for (const [vaultName, secretPath] of secretPaths) {
             await writer.write({
               nameOrId: vaultName,
               secretName: secretPath ?? '/',
-              metadata: first
-                ? { ...auth, options: { continueOnError: true } }
-                : undefined,
+              metadata: auth,
             });
-            first = false;
           }
           await writer.close();
           // Print out incoming data to standard out
           let hasErrored = false;
           for await (const chunk of response.readable) {
-            if (chunk.error) {
+            if (chunk.type === 'error') {
               hasErrored = true;
-              process.stderr.write(chunk.error);
+              // TypeScript cannot properly decode this type, so we need to use
+              // the `as` keyword so it doesn't throw errors.
+              const error = chunk as ErrorMessage;
```

Comment:
We need to use the `as` keyword in every `secrets` command which needs to continue on error, like `mkdir`, `cat`, etc. and this can cause a sprawl of the `as` keyword. The keyword should be used as little as possible, but this way, it is actually being used quite often. I'm not sure how good is this idea in the long term.